src/assets/icons/TwitterIcon.jsx

```diff
@@ -0,0 +1,9 @@
+export const TwitterIcon = () => (
```

Comment:
I'm amused that these are dynamic at all and not just static .svg files.

Reply:
They're components which can be embedded anywhere in pages/other components